saleor/graphql/order/mutations/orders.py

```diff
@@ -898,6 +908,9 @@ def save(cls, info, instance, cleaned_input):
                 code=OrderErrorCode.INSUFFICIENT_STOCK,
             )
         recalculate_order(instance.order)
+        transaction.on_commit(
```

Comment:
Are we sure that we want to trigger a webhook here? We can add/remove lines only when the order is in `DRAFT` and `UNCONFIRMED` status. This will trigger one webhook for any change in quantity number in a single line for `DRAFT` and for `UNCONFIRMED` order. The same situation is for `OrderLineCreate` and `OrderLineDelete`. 
cc @maarcingebala, @patrys

Reply:
@jakubkuc  Okay, everything is clear for me. The one thing which I think needs to be changed is that we should not call `order_updated` when this mutation is called for DRAFT order.  (And it is related to all mutations called on order lines create/update/delete).

Reply:
We could call the right plugin method based on order's status, couldn't we? I mean something like:
```
if order.status == OrderStatus.DRAFT:
    func = info.context.plugins.draft_order_updated
else:
    func = info.context.plugins.order_updated
```

Because now I believe we use the same mutation for updating order lines in drafts and regular orders.

Reply:
Yes, sorry,  I forgot to mention how we should solve it. As @maarcingebala  said, I think we can create simple `if else`